### PR TITLE
Fixing formatting & run instructions.

### PR DESCRIPTION
**Pull request by wiviw5**
This includes changes to README.md and a new page for RUNNING.md that will be updated more in the future as changes are made.

### CODE REVIEW

**Comment by wiviw5** [comment]
If you would like to preview the changes for yourself, you can at:

https://github.com/wiviw5/NerdBot/blob/fix_readmes/README.md
https://github.com/wiviw5/NerdBot/blob/fix_readmes/RUNNING.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```shell
+$ java -Dbot.token="DISCORD_BOT_TOKEN" -Dmongodb.uri="MONGODB_URI" -Dbot.environment="ENVIRONMENT" -Dbot.config="INSERT_CONFIG_PATH_HERE" -jar NerdBot.jar
+```
+
+
+## Config
+
+An example config file can be
+found [here](https://github.com/TheMGRF/NerdBot/blob/master/src/main/resources/example-config.json).
+
```
Comment (Aerhhh) [inline CONTRIBUTING.md:20]:
```suggestion

 - This file is automatically updated when values are added, removed, or changed.

```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+### For the most basic config to be valid it will need:
+
+- Emoji's all set (Agree, disagree, neutral, and greenlit)
```
Comment (Aerhhh) [inline CONTRIBUTING.md:23]: Make this into another bulleted list and change "Emoji's" to "Emojis"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- Log channel ID set
+- Guild ID set
+- Activity string set
+
+### Additionally, other things may be required to be set in the config before those features will work.
+
+- Suggestion forum id for suggestion parsing.
```
Comment (Aerhhh) [inline CONTRIBUTING.md:30]:
```suggestion
- Channel ID of a suggestions channel in order to curate suggestions
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- Item gen channel(s) for allowing itemgen to work in them.
```
Comment (Aerhhh) [inline CONTRIBUTING.md:31]:
```suggestion
- Channel ID('s) of item generator channel(s) to allow the item generation commands to be used
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- Tag ID set for greenlit to work.
```
Comment (Aerhhh) [inline CONTRIBUTING.md:32]: Not needed anymore since it checks by name instead of ID

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+### Other notes:
+
+Depending on the current environment, the configuration file will either be named `production.config.json`
+or `dev.config.json`
```
Comment (Aerhhh) [inline CONTRIBUTING.md:37]: Remove the newline

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+Changes to the configuration file require the bot to be restarted.
```
Comment (Aerhhh) [inline CONTRIBUTING.md:39]: There's a `/config reload` command to hot-reload the config while the bot is active

**Review by Aerhhh — CHANGES_REQUESTED** [review]

**Pull request merged**